tests/make_clean_db_with_grades.py

```diff
@@ -12,7 +12,7 @@
 from gluon.globals import Request
 
 # bring in the assignments controllers
-execfile("applications/runestone/controllers/assignments.py", globals())
+execfile("applications/runestone/modules/rs_grading.py", globals())
```

Comment:
Ahhh, so when paul moved a bunch of stuff out of assignments.py and into the rs_grading.py module that was probably the source of the breakage.

Reply:
Yep -- that's the root of the problem.